modules/aws_vpc/main.tf

```diff
@@ -34,25 +34,6 @@ resource "aws_vpc" "vpc" {
   )
 }
 
-resource "aws_vpc_endpoint" "private_endpoints" {
-  count              = length(var.vpc_private_endpoints)
-  vpc_id             = local.vpc_id
-  service_name       = "com.amazonaws.${var.region}.${var.vpc_private_endpoints[count.index]}"
-  vpc_endpoint_type  = "Interface"
-  security_group_ids = [ var.security_group_id ]
-
-  tags = merge(
-    {
-      "Name" = format("%s", "${var.name}-private-endpoint-${var.vpc_private_endpoints[count.index]}")
-    },
-    var.tags,
-  )
-
-  subnet_ids = [ 
-    for subnet in local.private_subnets : subnet.id
-  ]
-}
-
```

Comment:
This code was here in order to force the items into using the private_endpoints for everything internally regardless if it was a pubic/private cluster. What/how is this handled now? I see a lot of logic removed or these items below. This code kept the infra locked down to help in both the public/private scenarios.